src/service/poc_lora.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package helium.poc_lora;
+
+message poc_lora_beacon_req_v1 {
+  bytes poc_key = 1;
+  bytes pub_key = 2;  
+  bytes entropy = 3;
+  bytes data = 4;
+  uint64 timestamp = 5;
+  sint32 signal = 6;
+  float snr = 7;
+  float frequency = 8;
+  int32 channel = 9;
+  string datarate = 10;
+  bytes addr_hash = 11;
```

Comment:
addr_hash is no longer necessary, I don't think.

Reply:
addressed here: https://github.com/helium/proto/pull/154/commits/69497c85fbe35ea4f5ba6d79155d9629b21854b8